packages/server/database/migrations/20210622170418-addPokerStats.ts

```diff
@@ -0,0 +1,54 @@
+import {R} from 'rethinkdb-ts'
+import {MeetingTypeEnum, NewMeetingPhaseTypeEnum} from 'parabol-client/types/graphql'
+
+export const up = async function(r: R) {
+  try {
+    const getDiscussionIds = (meetingRow) => {
+      return meetingRow('phases')
+        .filter({phaseType: NewMeetingPhaseTypeEnum.ESTIMATE})('stages')
+        .concatMap((row) => row('discussionId'))
+    }
+    const getStories = (meetingRow) => {
+      return meetingRow('phases')
+        .filter({phaseType: NewMeetingPhaseTypeEnum.ESTIMATE})('stages')
```

Comment:
Makes sense. I've pushed an update so that the story count is calculated using complete stages but the commentCount is calculated using complete & incomplete stages